code/ZAS/Fire.dm

```diff
@@ -7,6 +7,9 @@ If it gains pressure too slowly, it may leak or just rupture instead of explodin
 */
 
 //#define FIREDBG
+#define FIRE_LIGHT_1	2 //These defines are the power of the light given off by fire at various stages
```

Comment:
done
(i'm just adding comments to keep track of what ive fixed, dont mind this)